xbmc/addons/kodi-dev-kit/include/kodi/c-api/addon-instance/inputstream/stream_crypto.h

```diff
@@ -100,6 +100,15 @@ extern "C"
 
     /// @brief Key id
     uint8_t kid[16];
+
+    /// @brief Encryption mode
+    uint16_t mode;
+
+    /// @brief Crypt blocks - number of blocks to encrypt in sample encryption pattern
+    uint8_t cryptBlocks;
+
+    /// @brief Skip blocks - number of blocks to skip in sample encryption pattern
+    uint8_t skipBlocks;
```

Comment:
You need to increase ADDON_INSTANCE_VERSION_INPUTSTREAM and ADDON_INSTANCE_VERSION_INPUTSTREAM_MIN by this change:
https://github.com/xbmc/xbmc/blob/b70c1598bbd0ec13263694554d262d964c97bd8d/xbmc/addons/kodi-dev-kit/include/kodi/versions.h#L111-L112
and create new releases about inpustream addons.

Otherwise newer addons bring crash in older Kodi.
Via binary addon repo, becomes addon faster updated as related Kodi then.

Reply:
Thanks @AlwinEsch 
Quick question - both to 3.0.3?

Reply:
Yes, 3.0.3 is good.

Reply:
Done :)

Reply:
Because it's backwards-incompatible, semantically I would bump the minor version.

Reply:
Sure thing